visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -131,7 +131,7 @@ MotionPlanningDisplay::MotionPlanningDisplay() :
   show_workspace_property_ = new rviz::BoolProperty("Show Workspace", false, "Shows the axis-aligned bounding box for the workspace allowed for planning",
                                                     plan_category_,
                                                     SLOT(changedWorkspace()), this);
-  query_start_state_property_ = new rviz::BoolProperty("Query Start State", true, "Shows the start state for the motion planning query",
+  query_start_state_property_ = new rviz::BoolProperty("Query Start State", false, "Set a custom start state for the motion planning query",
```

Comment:
On Sun, Jul 10, 2016 at 11:52:29PM -0700, Robert Haschke wrote:
> I don't see the value of disabling "Query Start State"...

Well, for one thing a collegue of mine would stop complaining that on each rviz startup he accidentally moves the start marker first, because start and goal marker overlap.
In the best case he notices the mistake before executing the resulting plan (which brings us back to the main issue)...
I've heard that story over and over from multiple users of this display and I've seen a number of students (including myself some years ago) having trouble using the display because of two interactive markers where everyone *using* the display expects just one, i.e. the goal position.
Therefore I believe the start state marker should be disabled by default.